main/character.py

```diff
@@ -2,5 +2,37 @@
 
 
 class Character:
-    def __init__(self):
-        pass
+    def __init__(self, x, y, name, strength, defense, health_points):
+        self.name = name
+        self.strength = strength
+        self.defense = defense
+        self.health_points = health_points
+        self.alive = True
+        self.image = pygame.image.load()
+        self.rect = self.image.get_rect()
+        self.rect.center = (x, y)
+
+    def attack(self, target):
```

Comment:
It might  be good idea to specify type of expected target (I think it always will be child class of character), it will be easier to use this in the future, because pycharm will highlight object if it will not match. Also it will predict methods of `target` in this function properly.
```suggestion
    def attack(self, target: Character):
```

Reply:
I will change it in next version :) Thx for tip!